src/extensions/scenario/scenarioext.cpp

```diff
@@ -158,6 +158,8 @@ void ScenarioClassExtension::Detach(TARGET target, bool all)
 void ScenarioClassExtension::Compute_CRC(WWCRCEngine &crc) const
```

Comment:
The `ScenarioClassExtension` constructor is missing initialisation of `IsIceDestruction`, which I assume should be set to `true`.